docs/OnTrack/UI Enhancement/ComponentReviewCreateUnitModal.md

```diff
@@ -0,0 +1,40 @@
+# Component Review
+
+## Student Name: Gaganjeet Singh
+
+## Student ID: 220032936
+
+## Term: T1 2023 – SIT 378
+
+## Ontrack Component review check
+
+First select a component to review from the list below:
+
+[https://deakin365.sharepoint.com/:x:/r/sites/ThothTech2/Shared%20Documents/OnTrack%20-%20UI%20Enhancement/T3_2022/Management%20%5BT3_2022%5D/selected_task(revised).xlsx?d=wac02013da5224c958ac60fd96fac7b20&csf=1&web=1&e=UfCmZ5](<https://deakin365.sharepoint.com/:x:/r/sites/ThothTech2/Shared%20Documents/OnTrack%20-%20UI%20Enhancement/T3_2022/Management%20%5BT3_2022%5D/selected_task(revised).xlsx?d=wac02013da5224c958ac60fd96fac7b20&csf=1&web=1&e=UfCmZ5>)
+
+## Component Name
+
+\*\*\*\*./src/app/admin/modals/create-unit-modal/create-unit-modal.coffee
+
+**File Name:**create-unit-modal.coffee
+
+## Component purpose
+
+## It is used to create a new unit
+
+## Component outcomes/interactions
+
+It creates a new unit. ![image](Images/CreateUnitModal.png)
+
+**Component migration Check list** – What is needs to be checked for this component to work once
+migrated?
+
+1. The modal for creating the unit is working.
+
+   Existing API endpoint can be used
+
+2. Add should have a dropdown for teaching period.
+
+   Update API to accommodate this extra field
+
+3. Needs redesign (use material design)
```

Comment:
We need a plan for the design now to make sure that it fits.

Reply:
I have attached a screenshot of the sketch for this one. I am not sure what you mean by 'plan for the design'.